src/migration/1617312788972-InitMigration.ts

```diff
@@ -0,0 +1,49 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class InitMigration1617312788972 implements MigrationInterface {
```

Comment:
I dont like the name please replace all date numbers lol

Reply:
na.... LGTM!